### PR TITLE
repo-sync-2024-08-27T17:40:19+0800

### PR DESCRIPTION
**Pull request by jiangzc86**

### CODE REVIEW

**Comment by jiangzc86** [comment]
I have read the CLA Document and I hereby sign the CLA

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,14 +14,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 `Fixed` for any bug fixes.
 `Security` in case of vulnerabilities.
 
-
```
Comment (oeqqwq) [inline CHANGELOG.md:17]: revert wrong change

Comment (jiangzc86) [reply CHANGELOG.md:17]: fixed

**Review by da-niao-dan — APPROVED** [review]
lgtm

**Review by warriorpaw — APPROVED** [review]
LGTM

**Pull request merged**